roles/smartgateway/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # defaults file for smartgateway
 size: 1
-container_image_path: quay.io/infrawatch/smart-gateway:latest
```

Comment:
Is there now default image used?

Reply:
Override is defined here for the CSV:  https://github.com/infrawatch/smart-gateway-operator/pull/44/files#diff-cf39b9e07be6d20fbef91685302e38cdR192-R193

Default is now in the task setup itself:  https://github.com/infrawatch/smart-gateway-operator/pull/44/files#diff-e46d2eb07eed6f2d5429e475cafe6e63R7-R9

Reply:
Ah missed that, thanks